python/whylogs/api/logger/experimental/logger/actor/thread_actor.py

```diff
@@ -57,14 +57,14 @@ def size(self) -> int:
 ThreadMessageType = TypeVar("ThreadMessageType")
 
 
-class ThreadActor(Actor, th.Thread, Generic[ThreadMessageType]):
+class ThreadActor(Actor[ThreadMessageType], th.Thread, Generic[ThreadMessageType]):
     """
     Subclass of Actor that uses a thread to process messages.
     """
 
     def __init__(self, queue_config: QueueConfig = QueueConfig()) -> None:
-        # our mypy version has a false positive on this super call
-        super().__init__(ThreadQueueWrapper[ThreadMessageType](), queue_config)  # type: ignore
+        # mypy can't infer but pyright can
```

Comment:
That suggests we don't need the `ignore`?

Reply:
The comment was misleading, I removed it. The real type_ignore happens because mypy doesn't like that the Thread init is invoked with these arguments through multiple inheritance, but the Thread init was designed to only look at certain kwargs and doesn't mind this.